### ISSUE TITLE
`Last Updated` displays UTC instead of local time under node v8.4.0

### ISSUE THREAD

**Issue by Krocko** [ebaauw/homebridge-hue#159]
In the homebridge Log the „last updated“ Time is 2 hours behind my local Time. 
`Sep 02 13:09:04 raspberrypi homebridge[7147]: [2017-9-2 13:09:04] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 0 to 1 on Sat Sep 02 2017 11:09:04`

Also the Home App is showing this wrong Time.

**Comment by ebaauw** [comment]
My guess is timezone settings.  The time reported by the Hue bridge should be in UTC, which homebridge-hue converts to local time.  You might want to check the output of `homebridge -D`:
```
[9/2/2017, 2:14:52 PM] [Hue] Living Room Temperature: sensor temperature changed from 2093 to 2107 on 2017-09-02T12:14:52
[9/2/2017, 2:14:52 PM] [Hue] Living Room Temperature: set homekit temperature from 20.9°C to 21.1°C on Sat Sep 02 2017 14:14:52
```

The first (debug) message (only shown for `-D`) shows the bridge (UTC) time; the second (info) message shows the (HomeKit) local time.

**Comment by Krocko** [comment]
My timezone is set to Europe/Berlin. The local Time is correct. The debug output is the same. The Sensor Time is still 2 hours behind.

**Comment by ebaauw** [comment]
> The debug output is the same.

The same as what?  Could you please post the messages?  And the messages with the versions (one for homebridge-hue and one for the Hue bridge)?

**Comment by Krocko** [comment]
What message do you mean?
```
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Philips hue: BSB002 bridge v1707040932, api v1.20.0
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] deCONZ-GW: deCONZ bridge v2.04.73, api v1.0.2
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] deCONZ-GW: 9 accessories
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Philips hue: 6 accessories
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Philips hue'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Philips hue: 3 services
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Pauls Lala'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Mikrowelle'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Abzug'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Fernsehsteckdose'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Vitrine'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'deCONZ-GW'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] deCONZ-GW: 2 services
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Stehlampe'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Badlampe'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Fernsehlampe'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Nachttischlampe'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Flurlampe'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Küchenlampe'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Bewegungsmelder Flur'...
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Initializing platform accessory 'Bewegungsmelder Küche'...
```
```
Sep 02 20:36:17 raspberrypi homebridge[8381]: [2017-9-2 20:36:17] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 0 to 1 on Sat Sep 02 2017 18:36:17
Sep 02 20:36:17 raspberrypi homebridge[8381]: [2017-9-2 20:36:17] [Philips Hue] Flurlampe: homekit power changed from 0 to 1
Sep 02 20:36:21 raspberrypi homebridge[8381]: [2017-9-2 20:36:21] [Philips Hue] Bewegungsmelder Küche: set homekit motion from 0 to 1 on Sat Sep 02 2017 18:36:21
Sep 02 20:36:22 raspberrypi homebridge[8381]: [2017-9-2 20:36:22] [Philips Hue] Küchenlampe: homekit power changed from 0 to 1Sep 02 20:36:31 raspberrypi homebridge[8381]: [2017-9-2 20:36:31] [Philips Hue] Bewegungsmelder Küche: set homekit motion from 1 to 0 on Sat Sep 02 2017 18:36:31Sep 02 20:36:39 raspberrypi homebridge[8381]: [2017-9-2 20:36:39] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 1 to 0 on Sat Sep 02 2017 18:36:39
Sep 02 20:37:21 raspberrypi homebridge[8381]: [2017-9-2 20:37:21] [Philips Hue] Küchenlampe: homekit power changed from 1 to 0
Sep 02 20:37:21 raspberrypi homebridge[8381]: [2017-9-2 20:37:21] [Philips Hue] Bewegungsmelder Flur: set homekit light level from 0.0001 lux to 1.4328 lux on Sat Sep 02 2017 18:37:21Sep 02 20:38:13 raspberrypi homebridge[8381]: [2017-9-2 20:38:13] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 0 to 1 on Sat Sep 02 2017 18:38:13
Sep 02 20:38:23 raspberrypi homebridge[8381]: [2017-9-2 20:38:23] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 1 to 0 on Sat Sep 02 2017 18:38:23
Sep 02 20:38:24 raspberrypi homebridge[8381]: [2017-9-2 20:38:24] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 0 to 1 on Sat Sep 02 2017 18:38:24
```

**Comment by ebaauw** [comment]
> What message do you mean?

These two, thanks
```
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] Philips hue: BSB002 bridge v1707040932, api v1.20.0
Sep 02 20:35:28 raspberrypi homebridge[8381]: [2017-9-2 20:35:28] [Philips Hue] deCONZ-GW: deCONZ bridge v2.04.73, api v1.0.2
```
And the message for homebridge-hue's version, a bit earlier in the log.  It looks like:
```
[9/2/2017, 8:45:08 PM] Loading 1 platforms...
[9/2/2017, 8:45:08 PM] [Hue] Initializing Hue platform...
[9/2/2017, 8:45:08 PM] [Hue] homebridge-hue v0.5.26, node v6.11.2, homebridge v0.4.27
```
So you have both a Hue bridge and a deCONZ gateway connected to homebridge-hue.  And it's the  sensors connected to deCONZ that show the wrong time?

> My timezone is set to Europe/Berlin.

Both in deCONZ and on the Raspberry?  For deCONZ, check _Show Advanced Settings_  under _Settings_ in the webapp (or check `/config/timezone` through the API).  For the Raspberry, run `sudo raspi-config` and check option 4 (Localisation Options), I2 (Change Timezone).

> The debug output is the same.

Looks like you're starting homebridge automatically.  In that case you need to add the `-D` flag to the script that starts  homebridge.  Alternatively, temporary disable the service (which should stop homebridge) and run `homebridge -D` from the command line.  You'll get an EADDRINUSE error if you try to run homebridge when it already running.

**Comment by Krocko** [comment]
> And the message for homebridge-hue's version, a bit earlier in the log. It looks like:
```
Sep  2 20:35:21 raspberrypi homebridge[8381]: [2017-9-2 20:35:21] [Philips Hue] Initializing Hue platform...
Sep  2 20:35:21 raspberrypi homebridge[8381]: [2017-9-2 20:35:21] [Philips Hue] homebridge-hue v0.5.26, node v8.4.0, homebridge v0.4.27
```
> So you have both a Hue bridge and a deCONZ gateway connected to homebridge-hue. And it's the sensors connected to deCONZ that show the wrong time?

Yes, right.
Before the sensors were connected to the hue bridge and the time was also wrong.

> Both in deCONZ and on the Raspberry?
Yes.

What part of the debug output do you need?
`[2017-9-2 21:13:48] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 0 to 1 on Sat Sep 02 2017 19:13:48[2017-9-2 21:14:06] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 1 to 0 on Sat Sep 02 2017 19:14:06`

**Comment by ebaauw** [comment]
What part of the debug output do you need?

I need the debug messages where homebridge-hue reports the `lastupdated` value on the bridge/gateway, like:
```
[9/2/2017, 2:14:52 PM] [Hue] Living Room Temperature: sensor temperature changed from 2093 to 2107 on 2017-09-02T12:14:52
```

**Comment by Krocko** [comment]
> I need the debug messages where homebridge-hue reports the lastupdated value on the bridge/gateway, like:

Yes, this i have posted:`[2017-9-2 21:13:48] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 0 to 1 on Sat Sep 02 2017 19:13:48[2017-9-2 21:14:06] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 1 to 0 on Sat Sep 02 2017 19:14:06`

**Comment by ebaauw** [comment]
No, these are the regular _info_ messages that homebridge-hue sets the HomeKit values.  I need the _debug_ messages without 'homekit" in them that show the bridge/gateway values.

**Comment by Krocko** [comment]
```
[2017-9-2 21:54:06] [Philips Hue] deCONZ-GW: hue bridge request #33: ok
[2017-9-2 21:54:06] [Philips Hue] Philips hue: hue bridge request #32: ok
[2017-9-2 21:54:07] [Philips Hue] Bewegungsmelder Küche: event: sensor presence changed from false to true on 2017-09-02T19:54:07
[2017-9-2 21:54:07] [Philips Hue] Bewegungsmelder Küche: set homekit motion from 0 to 1 on Sat Sep 02 2017 19:54:07
[2017-9-2 21:54:07] [Philips Hue] Küchenlampe: homekit power changed from 0 to 1
[2017-9-2 21:54:07] [Philips Hue] deCONZ-GW: hue bridge request #34: put /lights/6/state {"on":true}
[2017-9-2 21:54:07] [Philips Hue] deCONZ-GW: hue bridge request #34: ok
[2017-9-2 21:54:07] [Philips Hue] Philips hue: hue bridge request #33: get /config
[2017-9-2 21:54:07] [Philips Hue] deCONZ-GW: hue bridge request #35: get /config
[2017-9-2 21:54:08] [Philips Hue] deCONZ-GW: hue bridge request #35: ok
[2017-9-2 21:54:08] [Philips Hue] deCONZ-GW: hue bridge request #36: get /sensors
[2017-9-2 21:54:08] [Philips Hue] Philips hue: hue bridge request #33: ok
[2017-9-2 21:54:08] [Philips Hue] Philips hue: hue bridge request #34: get /sensors
[2017-9-2 21:54:08] [Philips Hue] deCONZ-GW: hue bridge request #36: ok
[2017-9-2 21:54:08] [Philips Hue] deCONZ-GW: hue bridge request #37: get /lights
[2017-9-2 21:54:08] [Philips Hue] deCONZ-GW: hue bridge request #37: ok
[2017-9-2 21:54:08] [Philips Hue] Philips hue: hue bridge request #34: ok
[2017-9-2 21:54:08] [Philips Hue] Philips hue: hue bridge request #35: get /lights
[2017-9-2 21:54:08] [Philips Hue] Philips hue: hue bridge request #35: ok
[2017-9-2 21:54:08] [Harmony Hub Wohnzimmer] debug Refreshing current activity
[2017-9-2 21:54:09] [Philips Hue] Philips hue: hue bridge request #36: get /config
[2017-9-2 21:54:09] [Philips Hue] deCONZ-GW: hue bridge request #38: get /config
[2017-9-2 21:54:10] [Philips Hue] deCONZ-GW: hue bridge request #38: ok
[2017-9-2 21:54:10] [Philips Hue] deCONZ-GW: hue bridge request #39: get /sensors
[2017-9-2 21:54:10] [Philips Hue] Philips hue: hue bridge request #36: ok
[2017-9-2 21:54:10] [Philips Hue] Philips hue: hue bridge request #37: get /sensors
[2017-9-2 21:54:10] [Philips Hue] deCONZ-GW: hue bridge request #39: ok
[2017-9-2 21:54:10] [Philips Hue] deCONZ-GW: hue bridge request #40: get /lights
[2017-9-2 21:54:10] [Philips Hue] Philips hue: hue bridge request #37: ok
[2017-9-2 21:54:10] [Philips Hue] Philips hue: hue bridge request #38: get /lights
[2017-9-2 21:54:10] [Philips Hue] deCONZ-GW: hue bridge request #40: ok
[2017-9-2 21:54:10] [Philips Hue] Philips hue: hue bridge request #38: ok
[2017-9-2 21:54:11] [Philips Hue] Bewegungsmelder Küche: event: sensor lightlevel changed from 0 to 0 on 2017-09-02T19:54:11
[2017-9-2 21:54:11] [Philips Hue] Philips hue: hue bridge request #39: get /config
[2017-9-2 21:54:11] [Philips Hue] deCONZ-GW: hue bridge request #41: get /config
[2017-9-2 21:54:12] [Philips Hue] deCONZ-GW: hue bridge request #41: ok
[2017-9-2 21:54:12] [Philips Hue] deCONZ-GW: hue bridge request #42: get /sensors
[2017-9-2 21:54:12] [Philips Hue] Philips hue: hue bridge request #39: ok
[2017-9-2 21:54:12] [Philips Hue] Philips hue: hue bridge request #40: get /sensors
[2017-9-2 21:54:12] [Philips Hue] deCONZ-GW: hue bridge request #42: ok
[2017-9-2 21:54:12] [Philips Hue] deCONZ-GW: hue bridge request #43: get /lights
[2017-9-2 21:54:12] [Philips Hue] Philips hue: hue bridge request #40: ok
[2017-9-2 21:54:12] [Philips Hue] Philips hue: hue bridge request #41: get /lights
[2017-9-2 21:54:12] [Philips Hue] deCONZ-GW: hue bridge request #43: ok
[2017-9-2 21:54:12] [Philips Hue] Philips hue: hue bridge request #41: ok
[2017-9-2 21:54:13] [Philips Hue] Philips hue: hue bridge request #42: get /config
[2017-9-2 21:54:14] [Philips Hue] deCONZ-GW: hue bridge request #44: get /config
[2017-9-2 21:54:14] [Philips Hue] deCONZ-GW: hue bridge request #44: ok
[2017-9-2 21:54:14] [Philips Hue] deCONZ-GW: hue bridge request #45: get /sensors
[2017-9-2 21:54:14] [Philips Hue] Philips hue: hue bridge request #42: ok
[2017-9-2 21:54:14] [Philips Hue] Philips hue: hue bridge request #43: get /sensors
[2017-9-2 21:54:14] [Philips Hue] deCONZ-GW: hue bridge request #45: ok
[2017-9-2 21:54:14] [Philips Hue] deCONZ-GW: hue bridge request #46: get /lights
[2017-9-2 21:54:14] [Philips Hue] Philips hue: hue bridge request #43: ok
[2017-9-2 21:54:14] [Philips Hue] Philips hue: hue bridge request #44: get /lights
[2017-9-2 21:54:14] [Philips Hue] deCONZ-GW: hue bridge request #46: ok
[2017-9-2 21:54:14] [Philips Hue] Philips hue: hue bridge request #44: ok
[2017-9-2 21:54:16] [Philips Hue] Philips hue: hue bridge request #45: get /config
[2017-9-2 21:54:16] [Philips Hue] deCONZ-GW: hue bridge request #47: get /config
[2017-9-2 21:54:16] [Philips Hue] deCONZ-GW: hue bridge request #47: ok
[2017-9-2 21:54:16] [Philips Hue] deCONZ-GW: hue bridge request #48: get /sensors
[2017-9-2 21:54:16] [Philips Hue] Philips hue: hue bridge request #45: ok
[2017-9-2 21:54:16] [Philips Hue] Philips hue: hue bridge request #46: get /sensors
[2017-9-2 21:54:16] [Philips Hue] deCONZ-GW: hue bridge request #48: ok
[2017-9-2 21:54:16] [Philips Hue] deCONZ-GW: hue bridge request #49: get /lights
[2017-9-2 21:54:16] [Philips Hue] Philips hue: hue bridge request #46: ok
[2017-9-2 21:54:16] [Philips Hue] Philips hue: hue bridge request #47: get /lights
[2017-9-2 21:54:16] [Philips Hue] deCONZ-GW: hue bridge request #49: ok
[2017-9-2 21:54:16] [Philips Hue] Philips hue: hue bridge request #47: ok
[2017-9-2 21:54:17] [Philips Hue] Bewegungsmelder Küche: event: sensor presence changed from true to false on 2017-09-02T19:54:17
[2017-9-2 21:54:17] [Philips Hue] Bewegungsmelder Küche: set homekit motion from 1 to 0 on Sat Sep 02 2017 19:54:17
[2017-9-2 21:54:18] [Philips Hue] Philips hue: hue bridge request #48: get /config
[2017-9-2 21:54:18] [Philips Hue] deCONZ-GW: hue bridge request #50: get /config
[2017-9-2 21:54:18] [Philips Hue] deCONZ-GW: hue bridge request #50: ok
[2017-9-2 21:54:18] [Philips Hue] deCONZ-GW: hue bridge request #51: get /sensors
[2017-9-2 21:54:18] [Philips Hue] Philips hue: hue bridge request #48: ok
[2017-9-2 21:54:18] [Philips Hue] Philips hue: hue bridge request #49: get /sensors
[2017-9-2 21:54:18] [Philips Hue] deCONZ-GW: hue bridge request #51: ok
[2017-9-2 21:54:18] [Philips Hue] deCONZ-GW: hue bridge request #52: get /lights
[2017-9-2 21:54:18] [Philips Hue] deCONZ-GW: hue bridge request #52: ok
[2017-9-2 21:54:18] [Philips Hue] Philips hue: hue bridge request #49: ok
[2017-9-2 21:54:18] [Philips Hue] Philips hue: hue bridge request #50: get /lights
[2017-9-2 21:54:18] [Philips Hue] Philips hue: hue bridge request #50: ok
[2017-9-2 21:54:20] [Philips Hue] Philips hue: hue bridge request #51: get /config
[2017-9-2 21:54:20] [Philips Hue] deCONZ-GW: hue bridge request #53: get /config
[2017-9-2 21:54:20] [Philips Hue] deCONZ-GW: hue bridge request #53: ok
[2017-9-2 21:54:20] [Philips Hue] deCONZ-GW: hue bridge request #54: get /sensors
[2017-9-2 21:54:20] [Philips Hue] Philips hue: hue bridge request #51: ok
[2017-9-2 21:54:20] [Philips Hue] Philips hue: hue bridge request #52: get /sensors
[2017-9-2 21:54:20] [Philips Hue] deCONZ-GW: hue bridge request #54: ok
[2017-9-2 21:54:20] [Philips Hue] deCONZ-GW: hue bridge request #55: get /lights
[2017-9-2 21:54:20] [Philips Hue] Philips hue: hue bridge request #52: ok
[2017-9-2 21:54:20] [Philips Hue] Philips hue: hue bridge request #53: get /lights
[2017-9-2 21:54:20] [Philips Hue] deCONZ-GW: hue bridge request #55: ok
[2017-9-2 21:54:20] [Philips Hue] Philips hue: hue bridge request #53: ok
[2017-9-2 21:54:22] [Philips Hue] Philips hue: hue bridge request #54: get /config
[2017-9-2 21:54:22] [Philips Hue] deCONZ-GW: hue bridge request #56: get /config
[2017-9-2 21:54:22] [Philips Hue] deCONZ-GW: hue bridge request #56: ok
[2017-9-2 21:54:22] [Philips Hue] deCONZ-GW: hue bridge request #57: get /sensors
[2017-9-2 21:54:22] [Philips Hue] Philips hue: hue bridge request #54: ok
[2017-9-2 21:54:22] [Philips Hue] Philips hue: hue bridge request #55: get /sensors
[2017-9-2 21:54:22] [Philips Hue] deCONZ-GW: hue bridge request #57: ok
[2017-9-2 21:54:22] [Philips Hue] deCONZ-GW: hue bridge request #58: get /lights
[2017-9-2 21:54:22] [Philips Hue] deCONZ-GW: hue bridge request #58: ok
[2017-9-2 21:54:22] [Philips Hue] Philips hue: hue bridge request #55: ok
[2017-9-2 21:54:22] [Philips Hue] Philips hue: hue bridge request #56: get /lights
[2017-9-2 21:54:22] [Philips Hue] Philips hue: hue bridge request #56: ok
[2017-9-2 21:54:24] [Philips Hue] Philips hue: hue bridge request #57: get /config
[2017-9-2 21:54:24] [Philips Hue] deCONZ-GW: hue bridge request #59: get /config
[2017-9-2 21:54:24] [Philips Hue] deCONZ-GW: hue bridge request #59: ok
[2017-9-2 21:54:24] [Philips Hue] deCONZ-GW: hue bridge request #60: get /sensors
[2017-9-2 21:54:24] [Philips Hue] Philips hue: hue bridge request #57: ok
[2017-9-2 21:54:24] [Philips Hue] Philips hue: hue bridge request #58: get /sensors
[2017-9-2 21:54:24] [Philips Hue] deCONZ-GW: hue bridge request #60: ok
[2017-9-2 21:54:24] [Philips Hue] deCONZ-GW: hue bridge request #61: get /lights
[2017-9-2 21:54:24] [Philips Hue] Philips hue: hue bridge request #58: ok
[2017-9-2 21:54:24] [Philips Hue] Philips hue: hue bridge request #59: get /lights
[2017-9-2 21:54:24] [Philips Hue] deCONZ-GW: hue bridge request #61: ok
[2017-9-2 21:54:24] [Philips Hue] Philips hue: hue bridge request #59: ok
[2017-9-2 21:54:26] [Philips Hue] Bewegungsmelder Flur: event: sensor presence changed from true to false on 2017-09-02T19:54:26
[2017-9-2 21:54:26] [Philips Hue] Bewegungsmelder Flur: set homekit motion from 1 to 0 on Sat Sep 02 2017 19:54:26
[2017-9-2 21:54:26] [Philips Hue] Philips hue: hue bridge request #60: get /config
[2017-9-2 21:54:26] [Philips Hue] deCONZ-GW: hue bridge request #62: get /config
[2017-9-2 21:54:26] [Philips Hue] deCONZ-GW: hue bridge request #62: ok
[2017-9-2 21:54:26] [Philips Hue] deCONZ-GW: hue bridge request #63: get /sensors
[2017-9-2 21:54:26] [Philips Hue] Philips hue: hue bridge request #60: ok
[2017-9-2 21:54:26] [Philips Hue] Philips hue: hue bridge request #61: get /sensors
[2017-9-2 21:54:26] [Philips Hue] deCONZ-GW: hue bridge request #63: ok
[2017-9-2 21:54:26] [Philips Hue] deCONZ-GW: hue bridge request #64: get /lights
[2017-9-2 21:54:26] [Philips Hue] Philips hue: hue bridge request #61: ok
[2017-9-2 21:54:26] [Philips Hue] Philips hue: hue bridge request #62: get /lights
[2017-9-2 21:54:26] [Philips Hue] deCONZ-GW: hue bridge request #64: ok
[2017-9-2 21:54:26] [Philips Hue] Philips hue: hue bridge request #62: ok
[2017-9-2 21:54:28] [Philips Hue] Philips hue: hue bridge request #63: get /config
[2017-9-2 21:54:28] [Philips Hue] deCONZ-GW: hue bridge request #65: get /config
[2017-9-2 21:54:28] [Philips Hue] deCONZ-GW: hue bridge request #65: ok
[2017-9-2 21:54:28] [Philips Hue] deCONZ-GW: hue bridge request #66: get /sensors
[2017-9-2 21:54:28] [Philips Hue] Philips hue: hue bridge request #63: ok
[2017-9-2 21:54:28] [Philips Hue] Philips hue: hue bridge request #64: get /sensors
[2017-9-2 21:54:28] [Philips Hue] Philips hue: hue bridge request #64: ok
[2017-9-2 21:54:28] [Philips Hue] Philips hue: hue bridge request #65: get /lights
[2017-9-2 21:54:28] [Philips Hue] deCONZ-GW: hue bridge request #66: ok
[2017-9-2 21:54:28] [Philips Hue] deCONZ-GW: hue bridge request #67: get /lights
[2017-9-2 21:54:28] [Philips Hue] deCONZ-GW: hue bridge request #67: ok
[2017-9-2 21:54:28] [Philips Hue] Philips hue: hue bridge request #65: ok
[2017-9-2 21:54:28] [Harmony Hub Wohnzimmer] debug Refreshing current activity
[2017-9-2 21:54:30] [Philips Hue] Philips hue: hue bridge request #66: get /config
[2017-9-2 21:54:30] [Philips Hue] deCONZ-GW: hue bridge request #68: get /config
[2017-9-2 21:54:30] [Philips Hue] deCONZ-GW: hue bridge request #68: ok
[2017-9-2 21:54:30] [Philips Hue] deCONZ-GW: hue bridge request #69: get /sensors
[2017-9-2 21:54:30] [Philips Hue] Philips hue: hue bridge request #66: ok
[2017-9-2 21:54:30] [Philips Hue] Philips hue: hue bridge request #67: get /sensors
[2017-9-2 21:54:30] [Philips Hue] deCONZ-GW: hue bridge request #69: ok
[2017-9-2 21:54:30] [Philips Hue] deCONZ-GW: hue bridge request #70: get /lights
[2017-9-2 21:54:30] [Philips Hue] Philips hue: hue bridge request #67: ok
[2017-9-2 21:54:30] [Philips Hue] Philips hue: hue bridge request #68: get /lights
[2017-9-2 21:54:30] [Philips Hue] deCONZ-GW: hue bridge request #70: ok
[2017-9-2 21:54:30] [Philips Hue] Philips hue: hue bridge request #68: ok
[2017-9-2 21:54:32] [Philips Hue] Philips hue: hue bridge request #69: get /config
[2017-9-2 21:54:32] [Philips Hue] deCONZ-GW: hue bridge request #71: get /config
[2017-9-2 21:54:32] [Philips Hue] deCONZ-GW: hue bridge request #71: ok
[2017-9-2 21:54:32] [Philips Hue] deCONZ-GW: hue bridge request #72: get /sensors
[2017-9-2 21:54:32] [Philips Hue] Philips hue: hue bridge request #69: ok
[2017-9-2 21:54:32] [Philips Hue] Philips hue: hue bridge request #70: get /sensors
[2017-9-2 21:54:32] [Philips Hue] deCONZ-GW: hue bridge request #72: ok
[2017-9-2 21:54:32] [Philips Hue] deCONZ-GW: hue bridge request #73: get /lights
[2017-9-2 21:54:32] [Philips Hue] Philips hue: hue bridge request #70: ok
[2017-9-2 21:54:32] [Philips Hue] Philips hue: hue bridge request #71: get /lights
[2017-9-2 21:54:32] [Philips Hue] deCONZ-GW: hue bridge request #73: ok
[2017-9-2 21:54:32] [Philips Hue] Philips hue: hue bridge request #71: ok
[2017-9-2 21:54:34] [Philips Hue] Philips hue: hue bridge request #72: get /config
[2017-9-2 21:54:34] [Philips Hue] deCONZ-GW: hue bridge request #74: get /config
[2017-9-2 21:54:34] [Philips Hue] deCONZ-GW: hue bridge request #74: ok
[2017-9-2 21:54:34] [Philips Hue] deCONZ-GW: hue bridge request #75: get /sensors
[2017-9-2 21:54:34] [Philips Hue] Philips hue: hue bridge request #72: ok
[2017-9-2 21:54:34] [Philips Hue] Philips hue: hue bridge request #73: get /sensors
[2017-9-2 21:54:34] [Philips Hue] deCONZ-GW: hue bridge request #75: ok
[2017-9-2 21:54:34] [Philips Hue] deCONZ-GW: hue bridge request #76: get /lights
[2017-9-2 21:54:34] [Philips Hue] Philips hue: hue bridge request #73: ok

```

**Comment by ebaauw** [comment]
Should be fixed in v0.5.27.

**Comment by Krocko** [comment]
Yes. Perfect! Thank you.
